### PR TITLE
test: add mongo db integration test

### PR DESCRIPTION
**Pull request by machi1990**
Fixes https://github.com/aerogear/graphback/issues/1562

- [x] fix failing tests

### CODE REVIEW

File: integration/.graphqlrc.yml [hunk 1]
```diff
@@ -18,7 +18,4 @@ extensions:
     ## Codegen plugins
     plugins:
```
Comment (machi1990) [inline integration/.graphqlrc.yml:19]: This should not be needed but kind of added it here to not ran through this issuehttps://github.com/aerogear/graphback/issues/1296

File: integration/tests/runtime-workflow-mongo.ts [hunk 2]
```diff
@@ -0,0 +1,437 @@
...
+
+    mongoClient = new MongoClient('mongodb://mongodb:mongo@localhost:27017/users?authSource=admin', { useUnifiedTopology: true });
+    await mongoClient.connect();
+    db = mongoClient.db('users');
+    graphbackApi = buildGraphbackAPI(modelText, {
+      dataProviderCreator: createMongoDbProvider(db),
+      plugins: [
+        new SchemaCRUDPlugin({outputPath: "./output-mongo/schema/schema.graphql"}),
+        new ClientCRUDPlugin({format: 'graphql', outputFile: './output-mongo/client/graphback.graphql'})
+      ]
```
Comment (machi1990) [inline integration/tests/runtime-workflow-mongo.ts:50]: Later on we could have the datasync plugin and add related tests here
/cc @ssd71

Comment (ssd71) [reply integration/tests/runtime-workflow-mongo.ts:50]: Sounds good

**Comment by machi1990** [comment]
For now the file https://github.com/aerogear/graphback/blob/5b5be08e03f361ebc389eaa038a137f4ba914da0/integration/tests/runtime-workflow-mongo.ts is just a replica of the postgres tests. But I think later on we could generate most of these tests and have them ran against different DB flavours we support.

File: integration/.graphqlrc.yml (hunk 1 above)
Comment (machi1990) [reply integration/.graphqlrc.yml:19]: Remove this hack once https://github.com/aerogear/graphback/pull/1569 is merged.

**Comment by wtrocki** [comment]
I would recomend to rename runtime-workflow to runtime workflow pg or something

**Comment by machi1990** [comment]
> I would recomend to rename runtime-workflow to runtime workflow pg or something

Good recommendation and this is already the case https://github.com/aerogear/graphback/blob/5b5be08e03f361ebc389eaa038a137f4ba914da0/integration/tests/runtime-workflow-postgres.ts

File: integration/.graphqlrc.yml [hunk 3]
```diff
@@ -15,10 +15,3 @@ extensions:
       subCreate: true
```
Comment (wtrocki) [inline integration/.graphqlrc.yml:15]: With that in mind we can also have crud config and location in js

**Comment by wtrocki** [comment]
Works for me.
@ssd71 comments?

File: integration/.graphqlrc.yml (hunk 3 above)
Comment (machi1990) [reply integration/.graphqlrc.yml:15]: To be fair, the file can be removed all together now. Let me do it.

Comment (machi1990) [reply integration/.graphqlrc.yml]: Done.

**Comment by ssd71** [comment]
Looks great! :smile:

**Pull request merged**